### PR TITLE
docs: javascript crypto identities

### PR DESCRIPTION
**Pull request by tpscrpt**
## Proposed changes

Update examples with `const { identities } = require('@arkecosystem/crypto)`

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [x] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] I have read the [WRITING DOCUMENTATION](https://docs.ark.io/guidebook/contribution-guidelines/writing-documentation.html) guidelines
- [x] I have added necessary documentation

### CODE REVIEW

**Review by faustbrian — CHANGES_REQUESTED** [review]
The examples should use the same addresses, public keys, private keys, WIFs and passphrases as here https://github.com/ArkEcosystem/docs/blame/master/docs/api/sdk/cryptography/php.md#L80.

**Comment by tpscrpt** [comment]
@faustbrian ok, changing now

File: docs/api/sdk/cryptography/javascript.md [hunk 1]
```diff
@@ -100,13 +100,13 @@ const { identities } = require('@arkecosystem/crypto')
...
 ```js
-identities.address.fromPassphrase('top secret passphrase')
+identities.address.fromPassphrase('this is a top secret passphrase')
 ```
 
 #### Get an address from a public key
 
 ```js
-identities.address.fromPublicKey(keys.publicKey)
+identities.address.fromPublicKey(keys.privateKey)
```
Comment (faustbrian) [inline docs/api/sdk/cryptography/javascript.md]: This should use the public key from the linked examples.

**Review by faustbrian — CHANGES_REQUESTED** [review]

**Comment by tpscrpt** [comment]
@faustbrian have changed, should be OK now

**Review by faustbrian — APPROVED** [review]

**Pull request merged**